Stage II/Single_Year_Stage_II.py

```diff
@@ -128,8 +124,13 @@ def Single_Year_Stage_II(puf, Stage_I_factors, Stage_II_targets, year, tol):
 
 
 
-    temp = [INTEREST,DIVIDEND, BIZ_INCOME, BIZ_LOSS, CAP_GAIN, ANNUITY_PENSION, SCH_E_INCOME, SCH_E_LOSS, SS_INCOME, UNEMPLOYMENT_COMP,
-            WAGE_1,WAGE_2, WAGE_3,WAGE_4, WAGE_5, WAGE_6, WAGE_7,WAGE_8,WAGE_9, WAGE_10, WAGE_11, WAGE_12]
+    temp = [Single, Joint, HH, SS_return, Dep_return,
+            INTEREST,DIVIDEND, BIZ_INCOME, BIZ_LOSS, CAP_GAIN, ANNUITY_PENSION,
+            SCH_E_INCOME, SCH_E_LOSS, SS_INCOME, UNEMPLOYMENT_COMP,
+            WAGE_1,WAGE_2, WAGE_3,WAGE_4, WAGE_5, WAGE_6,
+            WAGE_7,WAGE_8,WAGE_9, WAGE_10, WAGE_11, WAGE_12]
+    
+    b = []
     for m in temp:
         b.append(m)
```

Comment:
I still wasn't 100% sure on the difference between `temp` and `b`, but if you want a new list `b` that has the same contents as the list `temp`, I would do `b = list(temp)`. A minor change, but just something I noticed.